.circleci/config.yml

```diff
@@ -1,116 +1,17 @@
+# unused - remove file once circleCI integration is removed
```

Comment:
I might be mistaken, but should this file now be removed assuming it encompasses a full shift away from CircleCI?

Reply:
Originally I removed it, but then CircleCI shows up as a failed test with "no config file found" which I thought was distracting (and maybe even blocking the PR, not sure), so now I will remove the file once I removed that integration